### PR TITLE
Add Exercise05

### PR DESCRIPTION
**Pull request by huynhngocminh2511**

### CODE REVIEW

File: Exercise05/Program.cs [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+            Console.ReadKey();
+        }
+
+        static string ReadFile(string filePath)
+        {
+            var sr = new StreamReader(filePath);
+            var str = "";
+            var line = "";
+
+            while ((line = sr.ReadLine()) != null)
```
Comment (voquanghoa) [inline Exercise05/Program.cs]: You should use `StringBuilder` or `StringBuffer` instead

File: Exercise05/Program.cs [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+            Console.WriteLine(xmlString);
+
+            var xmldoc = new XmlDocument();
+            xmldoc.LoadXml(xmlString);
+            xmldoc.Save("../../../XmlFile/books.xml");
+
+            Console.ReadKey();
+        }
+
+        static string ReadFile(string filePath)
```
Comment (voquanghoa) [inline Exercise05/Program.cs]: Just need to use `File.ReadAllText(fileName)` (with import System.IO) is enought.

File: Exercise05/Program.cs [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+{
+    public class Program
+    {
+        public static string filePath = "../../../JsonFile/books.json";
+
+        static void Main(string[] args)
+        {
+            Console.Write("FilePath of JsonFile: ");
+            var str = Console.ReadLine();
+            if (!str.Equals(""))
```
Comment (voquanghoa) [inline Exercise05/Program.cs]: Use `!string.IsNullOrEmpty(str)`

File: Exercise05/Program.cs [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+            {
+                filePath = str;
+            }
+            var jsonString = ReadFile(filePath);
+
+            var obj = JsonConvert.DeserializeObject<List<Book>>(jsonString);
+            var xmlSerializer = new XmlSerializer(obj.GetType());
+
+            var strWriter = new StringWriter();
```
Comment (voquanghoa) [inline Exercise05/Program.cs]: Use `using`

File: Exercise05/Program.cs (hunk 3 above)
Comment (huynhngocminh2511) [reply Exercise05/Program.cs]: Done

File: Exercise05/Program.cs (hunk 4 above)
Comment (huynhngocminh2511) [reply Exercise05/Program.cs]: Done

File: Exercise05/Program.cs (hunk 2 above)
Comment (huynhngocminh2511) [reply Exercise05/Program.cs]: Done

File: Exercise05/Program.cs (hunk 1 above)
Comment (huynhngocminh2511) [reply Exercise05/Program.cs]: Done

**Pull request merged**